package/MDAnalysis/core/groups.py

```diff
@@ -1487,12 +1471,46 @@ class AtomGroup(GroupBase):
     AtomGroup instances are always bound to a
     :class:`MDAnalysis.core.universe.Universe`. They cannot exist in isolation.
 
+
+    .. rubric:: Deprecated functionality
+
+    *Instant selectors* will be removed in the 1.0 release.  See issue `#1377
+    <https://github.com/MDAnalysis/mdanalysis/issues/1377>`_ for more details.
+
+    Atoms can also be accessed in a Pythonic fashion by using the atom name as
```

Comment:
I confused the instant selectors with the Timeseries regarding "last release". But I still think that it should be documented for as long as it is available. 

(Otherwise the *deprecated:: 0.16.2* tag makes little sense: it says when it was deprecated, not when it was removed. The removal also kills the docs and possible leaves a *versionchanged:: 1.0.0* behind.)